Uplift/Views/Cells/GymWeekCell.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  GymWeekCell.swift
+//  Uplift
+//
+//  Created by Phillip OReggio on 10/6/19.
+//  Copyright © 2019 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+import SnapKit
+
+class GymWeekCell: UICollectionViewCell {
+
+    // MARK: - Views
+    private let dayLabel = UILabel()
+    private let backCircle = UIView()
+
+    // MARK: - Info
+    var weekDay: WeekDay!
+    private var isToday: Bool = false
+
+    override var isSelected: Bool {
+      didSet {
+        backCircle.alpha = isSelected || isToday ? 1 : 0
+        backCircle.backgroundColor = isSelected ? .fitnessYellow : .fitnessLightGrey
```

Comment:
bump on pulling master to get the new colors file -- these color names won't exist anymore when your colors file gets updated

Reply:
fixed in #190